src/components/TipCalc/TipCalc.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import './TipCalc.css';
+
+export default function TipCalc() {
+
+  const [percent, setPercent] = useState(25);
+  const [total, setTotal] = useState(0);
+
+  const calculateTip = () => (total * (percent / 100)).toFixed(2);
+  const calculateGrandTotal = () => (total * (percent / 100) + +total).toFixed(2);
+
```

Comment:
Hmm... I wonder if since you have two functions called calculateGrandTotal if you could pull them out of each component and put it in a .utils file? Pass it three arguments (total/percent/guests) and have some conditional logic for it to execute the correct equation

Reply:
I'm not familiar with .utils files. I'll look into it. Thanks!